tests/test-webapps/test-servlet-spec/test-spec-webapp/pom.xml

```diff
@@ -76,6 +76,7 @@
             <Import-Package>
               javax.transaction*;version="[1.1,1.3)",
               com.acme;version="${parsedVersion.majorVersion}.${parsedVersion.minorVersion}.${parsedVersion.incrementalVersion}",
+              org.eclipse.jetty.util;version="[9.3.0,10)",
```

Comment:
don't add this entry for jetty-util, see my comment on the issue.

Reply:
I am using the dependency plugin, like your comment asked.
This entry brings the manifest for the `test-spec-webapp.war` back to way it was for the 9.4.19 release.
See my comments on the manifest history.  I have 3 of them.   
First manifest shown is the jetty-9.4.x branch (aka what 9.4.19 release looks like).
Second is the manifest as it was when this PR was first created, and Greg pointed out this mistake.
Third is the manifest after adding this `Import-Package` line. (bringing the manifest back to where Greg wanted it)

Reply:
resolved.